pkgs/top-level/all-packages.nix

```diff
@@ -647,7 +647,9 @@ in
 
   aws_shell = pythonPackages.callPackage ../tools/admin/aws_shell { };
 
-  aws-sam-cli = callPackage ../development/tools/aws-sam-cli { };
+  aws-sam-cli = callPackage ../development/tools/aws-sam-cli {
+    python = python3;
```

Comment:
can we just use `python3` in the expression?

Reply:
sure